ls1intum/Artemis
# Bugfix/quiz exercises/space at start of lines

### Checklist

### Motivation and Context


Fixes #2569 

### Description

Added line indentation support for the short answer quiz exercises

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Navigate to Exercises
4. Create a quiz exercise with a short answer question
5. Add a code to the editor, such as the following:
```
`public class Foo {`
`    public `[-spot 1]`() {}`
`}`
[-option 1] Foo
```
6. Save
7. Check if the indentation is present in (pre)view mode.

### Test Coverage




### Screenshots



![image](https://user-images.githubusercontent.com/63168603/103575381-4dcc3e00-4eda-11eb-911a-900788f2770a.png)
